libcore/FileUtils.vala

```diff
@@ -615,25 +619,18 @@ namespace Files.FileUtils {
                 break;
         }
 
-        return get_formatted_date_time (dt, dateformatmode);
+        return get_formatted_date_time (dt, format);
     }
 
-    public string get_formatted_date_time (DateTime? dt, string? dateformatmode = null ) {
+    private string get_formatted_date_time (DateTime? dt, DateFormatMode format) {
         if (dt == null) {
             return "";
         }
 
-        string format;
-        if (dateformatmode == null) {
-            format = Files.Preferences.get_default ().date_format.down ();
-        } else {
-            format = dateformatmode;
-        }
-
-         switch (format) {
-            case "locale":
+        switch (format) {
+            case DateFormatMode.LOCALE:
                 return dt.format ("%c");
-            case "iso" :
+            case DateFormatMode.ISO :
```

Comment:
```suggestion
            case DateFormatMode.ISO:
```

We should unify if we have a space here with `case DateFormatMode.LOCALE:` and `default:`